File: src/main/java/me/muksc/tacztweaks/mixin/accessor/LocalPlayerDataHolderAccessor.java
package me.muksc.tacztweaks.mixin.accessor;

import com.tacz.guns.client.gameplay.LocalPlayerDataHolder;
import net.minecraft.client.player.LocalPlayer;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(value = LocalPlayerDataHolder.class, remap = false)
public interface LocalPlayerDataHolderAccessor {
    @Accessor("player")
    LocalPlayer tacztweaks$getPlayer();
}

File: src/main/kotlin/me/muksc/tacztweaks/mixininterop/LocalPlayerDataHolderAccessor.kt
package me.muksc.tacztweaks.mixininterop

import me.muksc.tacztweaks.mixin.accessor.LocalPlayerDataHolderAccessor
import net.minecraft.client.player.LocalPlayer

inline val LocalPlayerDataHolderAccessor.player: LocalPlayer
    get() = `tacztweaks$getPlayer`()